week-8/Homework/mandatory/2-fetch-exercise/exercise.js

```diff
@@ -17,10 +17,19 @@ Open index.html in your browser. Every time you refresh the page,
 a different greeting should be displayed in the box.
 */
 
-fetch('*** Write the API address here ***')
+fetch('https://codeyourfuture.herokuapp.com/api/greetings')
     .then(function(response) {
         return response.text();
     })
     .then(function(greeting) {
-        // Write the code to display the greeting text here
-    });
\ No newline at end of file
+
+        addgreetingToDOM(greeting);
+    })
+    .catch((error) => {
+        console.log(error);
+    })
+
+const addgreetingToDOM = (greeting) => {
+    const nameElement = document.getElementById('greeting-text');
+    nameElement.textContent = greeting;
+}
```

Comment:
Nice arrow function! You can immediately set the text content of the element without storing it in an interim variable because it is not used anywhere else.